akka-testkit/src/main/scala/akka/testkit/javadsl/TestKit.scala

```diff
@@ -292,8 +568,20 @@ class TestKit(system: ActorSystem) {
    * AssertionFailure being thrown in case of timeout.
    */
   @varargs
+  @Deprecated
+  @deprecated("Use the overloaded one which accepts java.time.Duration instead.", since = "2.5.12")
   def expectMsgAnyOf[T](max: FiniteDuration, objs: T*): T = tp.expectMsgAnyOf(max, objs: _*)
 
+  /**
+   * Receive one message from the test actor and assert that it equals one of
+   * the given objects. Wait time is bounded by the given duration, with an
+   * AssertionFailure being thrown in case of timeout.
+   */
+  def expectMsgAnyOf[T](max: java.time.Duration, objs: T*): T = {
+    import akka.util.JavaDurationConverters._
```

Comment:
would be ok to add this as an import at the top of the file to not have to do it at all places

Reply:
OK:)